modules/claims_api/lib/claims_api/error/error_handler.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Error
+    module ErrorHandler
+      def self.included(clazz)
+        clazz.class_eval do
+          rescue_from ::Common::Exceptions::TokenValidationError, with: :token_validation_error
+        end
+      end
+
+      private
+
+      def token_validation_error
+        json = {
```

Comment:
Rather than manually building json structures here, I think we should be building custom exception classes.  Something like `ClaimsApi::Error::TokenValidationError`.
Similar to what is done in this [dir](https://github.com/department-of-veterans-affairs/vets-api/tree/master/lib/common/exceptions) in `vets-api`.

I imagine this handler module is going to grow a fair amount over time and we don't want it to be littered with tons of manual JSON structure building.

Reply:
I moved it to it's own class. What do you think of the lambda bit, OK, or too much?

Thought: I don't really want to make a method for each type of error twice, so the lambda would take care of that.